x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/translations.ts

```diff
@@ -6,30 +6,40 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const SOURCE = i18n.translate('xpack.secops.networkTopNFlowTable.source.title', {
-  defaultMessage: 'Top Uni-Directional Flow Sources',
-});
-
-export const DESTINATION = i18n.translate('xpack.secops.networkTopNFlowTable.destination.title', {
-  defaultMessage: 'Top Uni-Directional Flow Destinations',
+export const TOP_TALKERS = i18n.translate('xpack.secops.networkTopNFlowTable.title', {
+  defaultMessage: 'Top Talkers',
 });
 
 export const SOURCE_IP = i18n.translate(
   'xpack.secops.networkTopNFlowTable.source.ip.column.title',
   {
-    defaultMessage: 'Source Ip',
+    defaultMessage: 'Source IP',
   }
 );
 
 export const DESTINATION_IP = i18n.translate(
   'xpack.secops.networkTopNFlowTable.destination.ip.column.title',
   {
-    defaultMessage: 'Destination Ip',
+    defaultMessage: 'Destination IP',
+  }
+);
+
+export const CLIENT_IP = i18n.translate(
+  'xpack.secops.networkTopNFlowTable.client.ip.column.title',
+  {
+    defaultMessage: 'Client IP',
+  }
+);
+
+export const SERVER_IP = i18n.translate(
+  'xpack.secops.networkTopNFlowTable.server.ip.column.title',
+  {
```

Comment:
Looking at the `i18n` guidelines and how the other code reviews have been going as well as other plugin code, title is usually not separated by a dot but rather it would be:

```ts
xpack.secops.networkTopNFlowTable.server.ip.column.serverTitle
```
Where it is suppose to end in camel case.